apm-sniffer/bootstrap-plugins/pom.xml

```diff
@@ -43,6 +43,7 @@
     <modules>
         <module>jdk-http-plugin</module>
         <module>jdk-threading-plugin</module>
+        <module>kotlin-coroutine-plugin</module>
```

Comment:
Why is kotlin coroutine plugin as bootstrap plugin? From my understanding, Kotlin still run inside JVM, and compiled as `.class` files. Bootstrap plugins should be rt.jar level injection. Does koltin plugin require this?

Reply:
I have another implementation for this plugin, it based on `ThreadContextElement` which is Kotlin coroutine context for syncing thread context, it needs to create interceptor point in `newCoroutineContext`, I got some bugs with creating the key object of `ThreadContextElement`(singleton be created twice), I change this plugin into bootstrap plugin, It works fine, I will do more test for it, and I will change it to a normal plugin after those test.